pybossa/jobs.py

```diff
@@ -958,11 +958,19 @@ def export_tasks(current_user_email_addr, short_name,
             with export_fn(project, ty, expanded, filters, disclose_gold) as fp:
                 filename = fp.filename
                 content = fp.read()
-            data = project.info
-            bucket_name = current_app.config.get('EXPORT_BUCKET')
-            max_size = current_app.config.get('EXPORT_MAX_SIZE', float('Inf'))
 
-            if len(content) > max_size and bucket_name:
+            bucket_name = current_app.config.get('EXPORT_BUCKET')
+            max_email_size = current_app.config.get('EXPORT_MAX_EMAIL_SIZE', float('Inf'))
+            max_s3_upload_size = current_app.config.get('EXPORT_MAX_UPLOAD_SIZE', float('Inf'))
+
+            if len(content) > max_s3_upload_size:
+                current_app.logger.info(f"task export exceeded max size. Project ID: {project.id}, Size: {len(content)}")
+                mail_dict['subject'] = 'Data export exceeded max file size: {0}'.format(project.name)
+                msg = '<p>Your export exceeded the maximum file upload size. ' + \
+                    'Please try again with a smaller subset of tasks or ' + \
+                    'reach out to a {0} administrator for help.</p>'
```

Comment:
reaching out to admin would be overkill if system imposes certain limits and its advisable to stick to smaller chunks of data